packages/wp-dashboard/src/api/story.js

```diff
@@ -165,12 +163,7 @@ export const createStoryFromTemplate = async (config, template) => {
     title: '',
   };
 
-  const content = getStoryMarkup(story, pages, {
-    publisher: createdBy,
```

Comment:
Does this mean that we're not using the `createdBy` field of templates anymore?

Reply:
We use it on the "Explore Templates" page to display who created the template.